addusers.py

```diff
@@ -388,11 +389,22 @@ def parse_rows(rows):
                    ks_users.append(new_ks_user)
                    subscribe_emails.append(user.email)                
                 elif not ks_user:
-                    raise ItemNotFoundError('User', user.name)
+                    if user.is_requestor:
+                        raise ItemNotFoundError('User', user.name)
+                    else:
+                        # We don't treat this as a critical error
```

Comment:
We care whether the person who filled out the form said they had an existing account, but doesn't.  

We don't actually care so much if someone they listed as an additional user doesn't exist.  That's much easier to fix later, so we just output a notice that we are skipping that person.